src/helpers/CSSVariablesHelper.tsx

```diff
@@ -0,0 +1,34 @@
+let cssVariables: {
+    [k: string]: string;
+} = {};
+
+let allCSS = [].slice
```

Comment:
This is just for the stories, right?

Reply:
I think you can just directly call `[].slice....` to start off the process without having to assign it to a variable you're not going to use.

I also suggest to make the inner `[].slice` call a separate function for readability.